postgres_mixin/alerts/postgres.libsonnet

```diff
@@ -120,6 +120,207 @@
               severity: 'warning',
             },
           },
+          {
+            alert: 'PostgresHasTooManyRollbacks',
+            annotations: {
+              description: 'PostgreSQL has too many rollbacks on {{ $labels.cluster }} for database {{ $labels.datname }} with a value of {{ $value }}',
+              summary: 'PostgreSQL has too many rollbacks.',
+            },
+            expr: |||
+              avg without(pod, instance)
+              (rate(pg_stat_database_xact_rollback{db_name!~"template.*|^$"}[5m]) /
```

Comment:
Question/ potention nit:

Why is the regex used here different to other queries above?
e.g: `PostgreSQLCacheHitRatio` uses `datname!~"template.*",`

However, this new line uses a separate label `db_name`, which doesn't match what is used in the description, and a selector of `template.*|^$` instead.

What's the difference here, and why the mismatch between `$labels.datname` and `db_name`?

Also, this is a general thought, maybe we should move regularly uses regexes like these to the configuration or a hardcoded variable defined once at the top of the file to prevent potential typos?